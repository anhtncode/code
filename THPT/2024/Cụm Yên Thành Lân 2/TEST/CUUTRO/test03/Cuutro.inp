9 3
6 6 1 8 4 9 6 3 7 
1 7 3
6 8 4
2 7 9
