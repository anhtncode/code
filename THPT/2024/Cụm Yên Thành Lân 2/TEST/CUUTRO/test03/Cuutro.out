9 18 13 20 16 25 22 7 7 